embedded/imu_driver/imu_balsaq.cc

```diff
@@ -86,13 +85,15 @@ void ImuBq::loop() {
     if (msg.accel_mpss_x == 0 && msg.accel_mpss_y == 0 && msg.accel_mpss_z == 0) {
       gonogo().nogo("IMU reading all zeroes (We will attempt a reset)!");
       all_go = false;
+      go_transmitted_ = false;
```

Comment:
I suggest doing this inside of gonogo itself -- only call `publish` when the state has changed. This way, BQ's don't need to know how/when the gonogo stuff is really publishing

Reply:
[Blocking only on this -- it'll be less bug-prone too to do it in `gonogo.cc`]

Reply:
Updated, now we only publish on state changes in the `gonogo` object.